app/jobs/shrine/promotion_job.rb

```diff
@@ -6,7 +6,17 @@ class Shrine::PromotionJob < ApplicationJob
   def perform(record:, name:, file_data:)
     attacher = Shrine::Attacher.retrieve(model: record, name: name.to_sym, file: file_data)
     attacher.atomic_promote
+    Shrine::ThumbnailJob.perform_later(record) if perform_thumbnail?(record)
   rescue Shrine::AttachmentChanged, ActiveRecord::RecordNotFound
     # attachment has changed or record has been deleted, nothing to do
   end
+
+  def perform_thumbnail?(record)
```

Comment:
This might be better in the FileResoruce as something like `thumbnailable?` and we could use it in IRB, rake tasks etc. ?

Reply:
Yeah, I'll see if I can move it there.

Reply:
I can, too -- just putting this out there for convo -- i wrote this silly code 🎉